### DISCUSSION TITLE
Store information from an external OIDC source's user data endpoint?

### DISCUSSION THREAD

**Discussion by thetayloredman** [goauthentik/authentik#8992] [Q&A]
I'm logging users in with Discord and I need a way for an external script to use the Authentik API and look up users by email and retrieve their Discord user ID. Is this possible?

**Comment by rissson** [comment]
We provide an API browser that you can use to find endpoints you need. You probably want to look at https://docs.goauthentik.io/developer-docs/api/browser#tag/core/operation/core_users_list and https://docs.goauthentik.io/developer-docs/api/browser#tag/sources/operation/sources_user_connections_oauth_list

**Reply by thetayloredman** [reply]
What property in oauth_retrieve/list is what I want?

**Reply by rissson** [reply]
`identifier`
